frontend/admin/src/js/components/pool/PoolTable.tsx

```diff
@@ -43,11 +50,25 @@ function poolCandidatesLinkAccessor(
 function viewLinkAccessor(
   editUrlRoot: string,
   id: string,
-  title: string | undefined | null,
+  title: Maybe<string>,
+  classifications: AccessorClassifications,
+  stream: PoolStream | null,
+  intl: IntlShape,
 ) {
+  let classificationString = ""; // type wrangling the complex type into a string
+  if (classifications && classifications[0]) {
+    const grouping = classifications[0];
+    classificationString = `${grouping.group}-0${grouping.level}`;
+  }
```

Comment:
Hmm, not sure if this is within the scope of this issue, but i feel like we could make this a helper too. We are doing something very similar in `PoolCard`
https://github.com/GCTC-NTGC/gc-digital-talent/blob/966c19094a9f3e7c825c0e36921526320b3d38a7/frontend/talentsearch/src/js/components/Browse/PoolCard/PoolCard.tsx#L30-L40

Reply:
sure why not, https://github.com/GCTC-NTGC/gc-digital-talent/pull/4533/commits/a4a670641de8e463b3f9d4179820272185c9621e